lib/cc/pull_requests.rb

```diff
@@ -1,8 +1,28 @@
 class CC::PullRequests < CC::Service
+  ABLE_TO_UPDATE_STATUS_MESSAGE = "Access token is valid"
```

Comment:
:lipstick: this is probably a missing/different rubocop config in this project, but we prefer to `freeze` mutable objects assigned to constants.